setup.py

```diff
@@ -92,6 +92,8 @@
         'pyserial',  # Serial port interface
         'wrapt',  # Basic for construction of decorator functions
         'future', # Python 2-3 compatibility
+        'pandas',
+        'numpy',
```

Comment:
I added these as dependencies as they are used by `BaylibreAcmeInstrument`; is this fine? Should they be in `extras_require`?

Reply:
Good question, I think for now they should probably go into ``extras_require`` and guarded against as well.

Reply:
Although I'm not sure where this code is being used, I see that `devlib/derived/fps.py` is `import`ing `pandas` so it seems like the dependency already existed in the code but wasn't expressed in `setup.py`?

I am a bit confused: from [the documentation](http://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies) (I've never used this particular parameter to `setup` myself), if they are in `extras_require`, we have to define [*entry points*](http://setuptools.readthedocs.io/en/latest/setuptools.html#entry-points) which become the keys to the `dict` passed as the `extras_require` argument and then "trigger" these points in the source (using `pkg_resources`? I'm not so sure ...), right?

But I don't see such calls in the codebase for the existing dependencies (`'daq'`, `'doc'` and `'monsoon'`) so I'm not really sure how I should implement this using the "proper" tools ...